src/styles/globals.css

```
@@ -4,9 +4,49 @@
+    var(--font-space-grotesk), "Space Grotesk", system-ui, sans-serif;
+  --sc-font-mono: var(--font-jb-mono), "JetBrains Mono", monospace;
+
+  /* Motion */
+  --sc-ease-out: cubic-bezier(0.16, 1, 0.3, 1);
+  --sc-ease-in-out: cubic-bezier(0.76, 0, 0.24, 1);
+  --sc-duration-fast: 200ms;
+  --sc-duration-med: 400ms;
+  --sc-duration-slow: 800ms;
+}
+
 @theme inline {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)